packages/client/src/object/convertWireToOsdkObjects/createOsdkObject.ts

```diff
@@ -127,6 +138,20 @@ export function createOsdkObject<
               p as string,
             );
           }
+          if (propDef.type === "geotimeSeriesReference") {
+            const instance = target[CachedPropertiesRef].get(p);
+            if (instance != null) {
+              return instance;
+            }
+            const geotimeProp = new GeotimeSeriesPropertyImpl<GeoJSON.Point>(
```

Comment:
I will do a FLUP for the other properties above to be cached, I wanted to keep this PR just to geotime stuff rn